command/agent/exec/exec.go

```diff
@@ -216,6 +223,21 @@ func (s *Server) Run(ctx context.Context, incomingVaultToken chan string) error
 				return fmt.Errorf("template server: %w", err)
 			}
 
+			// Calculate the amount of time to backoff using exponential backoff
+			sleep, err := restartBackoff.Next()
+			if err != nil {
+				ts.logger.Error("template server: reached maximum number restart attempts")
+				restartBackoff.Reset()
+			}
+
+			// Sleep for the calculated backoff time then attempt to create a new runner
+			ts.logger.Warn(fmt.Sprintf("template server restart: retry attempt after %s", sleep))
```

Comment:
I'm a little confused why this is repeated -- is this a copy/paste error?

Reply:
Ooops yes copy and paste error. Removing now.